lib/routes/email.js

```diff
@@ -17,14 +22,25 @@ module.exports = {
     }
   },
   handler: function email(req, reply) {
-    var email = req.auth.credentials.email;
-    if (email) {
+    request.get(AUTH_SERVER_URL, {
+      headers: {
+        Authorization: 'Bearer ' + req.auth.token
+      },
+      json: true
+    }, function(err, res, body) {
+      if (err) {
+        logger.error('request.auth_server.network', err);
+        return reply(new AppError.authError('network error'));
+      }
+      if (res.statusCode >= 400) {
+        logger.error('request.auth_server.fail', { code: res.statusCode });
+        return reply(new AppError.authError('auth server error'));
```

Comment:
I wish we could reflect through the error number form the auth-server here, but I guess that doesn't make much sense since auth and profile API errornos are different.